src/processes/news-process.js

```diff
@@ -1,213 +1,250 @@
         */
-        const getDomain = await validateDomain(crawls);
-        const domain_page_id = getDomain.domain_id;
-        const name = getDomain.name;
-        const info = getDomain.info;
-        const logo = getDomain.domain_image;
-        const created_domain = getDomain.created_domain;
-
-        /*
+    const getDomain = await validateDomain(crawls);
+    const domain_page_id = getDomain.domain_id;
+    const {name, info, created_domain, domain_image: logo} = getDomain;
+    let contentPublishedAt = null;
+
+    /*
           @description crawls attribut opengraph url domain
         */
-        const $ = cheerio.load(crawls.data);
-        const site_name = $('meta[property="og:site_name"]').attr('content') || "";
-        const title = $('meta[property="og:title"]').attr('content') || "";
-        const image = $('meta[property="og:image"]').attr('content') || "";
-        let description = $('meta[property="og:description"]').attr('content') || "";
-        // if (description === "") {
-        //   description = $('meta[name="description"]').attr('content') || "";
-        // }
-        const news_url = $('meta[property="og:url"]').attr('content') || "";
-        const url_compact = $('meta[property="og:url"]').attr('content') || "";
-        const keyword = $('meta[name="keywords"]').attr('content') || "";
-        const author = $('meta[name="author"]').attr('content') || "";
-
-        /*
+    const $ = cheerio.load(crawls.data);
+
+    const site_name = $('meta[property="og:site_name"]').attr('content') || '';
+    const title = $('meta[property="og:title"]').attr('content') || '';
+    const image = $('meta[property="og:image"]').attr('content') || '';
+    const description = $('meta[property="og:description"]').attr('content') || '';
+    // if (description === "") {
+    //   description = $('meta[name="description"]').attr('content') || "";
+    // }
+    const news_url = $('meta[property="og:url"]').attr('content') || '';
+    const url_compact = $('meta[property="og:url"]').attr('content') || '';
+    const keyword = $('meta[name="keywords"]').attr('content') || '';
+    const author = $('meta[name="author"]').attr('content') || '';
+    try {
+      const ldJson = $("script[type='application/ld+json']")[0]?.children?.at(0)?.data;
+      const ldParsedJson = JSON.parse(ldJson);
+      contentPublishedAt = ldParsedJson?.datePublished;
+    } catch (e) {
+      console.error(e);
+    }
+
+    /*
           @description define data for post to getstream and newslink table
         */
-        let data = {
-            domain_page_id, title, site_name, image, description, news_url, keyword, author, url_compact
-        };
-
-
-        /*
+    const data = {
+      domain_page_id,
+      title,
+      site_name,
+      image,
+      description,
+      news_url,
+      keyword,
+      author,
+      url_compact,
+      contentPublishedAt
+    };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the property `contentPublishedAt` is being assigned to `contentPublishedAt`. However, in the rest of the code, the camelCase version `content_published_at` is used. For consistency, it's better to stick with one naming convention.
```diff
-       contentPublishedAt
+       content_published_at: contentPublishedAt
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
will change

Reply:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

@fajarism Great! Consistency in naming conventions is crucial for code readability and maintainability. If you need further assistance, feel free to ask.

<!-- This is an auto-generated reply by OSS CodeRabbit -->